src/platform-admin/domain/authorization/admin.authorization.resolver.mutations.ts

```diff
@@ -43,7 +43,7 @@ export class AdminAuthorizationResolverMutations {
     @Inject(WINSTON_MODULE_NEST_PROVIDER)
     private readonly logger: LoggerService,
     private authResetService: AuthResetService,
-    private aiPersonaService: AiPersonaService,
+    private aiPersona: AiPersonaService,
```

Comment:
To be called aiPersonaService - all our other services have Service as a suffix. I know we probably did not have it to avoid ServiceService showing up.